guides/beginners_guide_adding_devices.rst

```diff
@@ -0,0 +1,704 @@
+
+.. note::
+
+    ESPHome devices usually rely on Home Assistant.  It's more than just an excellent way to control and automate your
+    home. It can also manage many IoT devices.  Read more about `Home Assistant <https://www.home-assistant.io/>`__.
+
+    If you are already running Home Assistant, check out :doc:`getting_started_hassio` to run ESPHome as an add-on.
+    
+    If running Home Assistant in Docker, get ESPHome by following instructions in the first part of
+    :doc:`getting_started_command_line`.
+
+    Or if not running Docker or Home Assistant at all, you can try :doc:`beginners_guide_docker_desktop`.
+
+Setting up your Secrets
+-----------------------
+
+A ``Secrets`` file can be used to hold things that you wish to remain secret. So, you can copy and paste a YAML
+from your own code to share with others, without the worry of leaking details about your home network, such as your
+SSID name and password.
+
+These elements can be re-used in all of your YAMLs, so it also makes copying and pasting YAML code from other
+sources (such as Github) much easier as well.
+
+Click here in the ESPHome Dashboard to edit ``Secrets``.
+
+.. figure:: images/noob_dashboard_1a.png
+    :align: center
+    :width: 95.0%
+    :alt: ESPHome Dashboard window with circle around "Secrets"
+
+Wi-Fi Network
+*************
+
+Add the next 4 lines, replacing the placeholders with your own values.
+
+.. code-block:: yaml
+
+    # My Wi-Fi SSID and password
+    wifi_ssid: "<MY_WIFI_NAME>"
+    wifi_password: "<MY_WIFI_PASSWORD>"
+    ap_password: "<MY_AP_PASSWORD>" # Super-secret!
+
+These are the 3 most basic secrets. The wireless network name is set by ``wifi_ssid`` and the password 
+is set by ``wifi_password``. Actually, the names ``wifi_ssid``, ``wifi_password`` are arbitrary.
+You could just as easily use ``home_wifi``, ``home_password`` if you like.
+But ``wifi_ssid``, ``wifi_password``, and ``ap_password`` are commonly used in the community.
+
+Finally, ``ap_password`` sets a password for the hotspot that an ESPHome device creates when it cannot
+find the wireless network.
+
+.. note::
+
+    Please note in the above example, I have included comments with the ``#`` character.  Comments can be
+    added as their own lines or at the end of lines.
+    Anything in the line after the ``#`` are ignored.
+    Often, comments can be used to explain what a particular section of code is doing.
+
+.. warning::
+
+    Although it is theoretically possible for an ESP-based device to use a 5Ghz or 6Ghz wireless network,
+    there are not many ESP-based devices that actually support those ranges.
+    Most ESP-based devices only support the 2.4Ghz range.
+    Although you should have no issues using the same SSID name for 2.4Ghz and 5Ghz, if you encounter any issues
+    with a device maintaining its connection, you may want to consider using a different SSID name
+    or guest network on the 2.4Ghz band for your IOT devices.
```

Comment:
Let's be very careful with our words here so as to not create any confusion. Theoretically possible, yes, but..."not many"? Other than the C5--which is brand-new and I'm not even sure is readily available yet--which ESPs support frequencies other than 2.4 GHz? 😇

```suggestion
    Supported Espressif microcontrollers only support the 2.4 Ghz frequency range for Wi-Fi.

    If you're experiencing connectivity issues, see our :ref:`FAQ <wifi-problems>`.
```